src/main/java/de/triology/cas/services/GetDoguNodeFromEtcdException.java

```diff
@@ -0,0 +1,4 @@
+package de.triology.cas.services;
+
+class GetDoguNodeFromEtcdException extends Throwable {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this class to remove "Exception" or correct its inheritance. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS2166)